06-building-a-program-start-to-finish/writing-tests/output/invertedIndexTests.js

```diff
@@ -0,0 +1,308 @@
+const invertedIndex = new InvertedIndex();
+describe('Test if the object exist and new instances can be created from it',
+ () => {
+   it('should return "function" when its type is checked', () => {
+     expect(typeof (InvertedIndex)).toEqual('function');
+   });
+
+   it('should create an object once the class is declared', () => {
+     expect(invertedIndex).toEqual(jasmine.any(Object));
+   });
+
+   it('should create an instance of itself once its attached to the' +
+  '"new" key word',
+  () => {
+    expect(invertedIndex instanceof InvertedIndex).toBeTruthy();
+  });
+ });
+
+describe('Test the checkForIndex functionality',
+ () => {
+   let firstContent; let secondContent; let filteredContents;
```

Comment:
I'm sorry! What's going on here?

Please please and please go READ the style guides!